src/command_modules/azure-cli-batchai/HISTORY.rst

```diff
@@ -0,0 +1,9 @@
+.. :changelog:
+
+Release History
+===============
+
+0.1.0 (2017-10-01)
```

Comment:
Please just leave `unreleased` here. Our release automation process will fill in the blank on the release day.

Reply:
done